src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: $border-size dotted $border-color;
+  border-radius: $circle;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__middle {
+    width: $middle-size;
+    height: $middle-size;
+    background-color: $middle-color;
+    border-radius: $circle;
+  }
+  &__minutes-hand {
+    position: absolute;
+    width: $hand-minutes-width;
+    height: $hand-minutes-height;
+    background-color: $hand-minutes-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-minutes-width} / 2));
+    z-index: -1;
+    animation: rotate-animation $minutes-duration steps(60) infinite;
+    transform-origin: bottom center;
+  }
+  &__seconds-hand {
+    width: $hand-seconds-width;
+    height: $hand-seconds-height;
+    background-color: $hand-seconds-color;
+    position: absolute;
+    z-index: -1;
```

Comment:
Same as the previous comment, consider grouping position, bottom, left, and z-index properties together.